### PR TITLE
Library v9.0.4 manifest with disabled archiving for @PlatformIO

### PR DESCRIPTION
**Pull request by ivankravets**
Resolves:
- https://github.com/stm32duino/STM32FreeRTOS/issues/17
- https://github.com/platformio/platform-ststm32/issues/237

### CODE REVIEW

File: library.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "keywords": "rtos, timing, thread, task, mutex, semaphore",
+    "description": "Real Time Operating System implemented for STM32",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/stm32duino/STM32FreeRTOS.git"
+    },
+    "frameworks": "arduino",
+    "platforms": "ststm32",
+    "build": {
+        "libArchive": false
```
Comment (fpistm) [inline library.json]: Do you advise to set this for each library?
I've the same PR on another library which does not specify this? is this correct?
Thanks in advance

Comment (ivankravets) [reply library.json]: If a library has `weak` attributes, then is better to disable archiving.

Comment (fpistm) [reply library.json]: Ok. Thanks.

**Comment by ivankravets** [comment]
Thanks!

**Comment by fpistm** [comment]
welcome. I will try to add this file on all STM32duino libraries when work on them.

**Comment by ivankravets** [comment]
Do you know someone from STMicro Team who can help us to add PlatformIO on this page https://www.st.com/en/development-tools/stm32-ides.html ?

**Comment by fpistm** [comment]
I guess the simplest way is to ask to ST community.
Anyway I will try to find who internally.

**Comment by ivankravets** [comment]
> Anyway I will try to find who internally.

Thanks!

**Comment by fpistm** [comment]
Hi @ivankravets 

> Do you know someone from STMicro Team who can help us to add PlatformIO on this page https://www.st.com/en/development-tools/stm32-ides.html ?

Sorry for the late answer. I've finally got the way to be referenced. 😭 
You should log in a request to join ST Partner by clicking “Join the program” at https://www.st.com/content/st_com/en/partner/partner-program.html

**Pull request merged**